metaworld/envs/mujoco/sawyer_xyz/sawyer_push_wall_v2.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
```

Comment:
@adibellathur please include a module docstring that describes this task/envrionment, along with the major changes between v1 and v2. If there arent any changes, then please write that there are no changes at the moment.

Reply:
Docstring added.